src/shared/bootspec.h

```diff
@@ -5,7 +5,8 @@
 #include <stdlib.h>
 
 typedef struct BootEntry {
-        char *filename;
+        char *filename; /* This is file basename without extension */
+        char *filepath; /* This is full path to the file */
```

Comment:
hmm, so #10175 changes this all, and cleans it up, please, let's merge that one first!